### PR TITLE
update to 0.11.0, use new upstream, modernize recipe

### LINKED ISSUE #1: Upstream was renamed to locustio
Issue:
It appears the upstream pypi package was renamed to `locustio`:

https://pypi.org/project/locustio/#history

Will PR applying the name in-place, but a new feedstock might be more appropriate?

### PR DESCRIPTION
**Pull request by nrbgt**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

closes #1

### CODE REVIEW

**Comment by nrbgt** [comment]
@conda-forge-admin please rerender

**Comment by nrbgt** [comment]
@carlodri hooray CI :green_heart:

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 93404f831114791b0756325c53b08bff73f048eeb69688be657629feaa62b507
 
 build:
   noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
```
Comment (carlodri) [inline recipe/meta.yaml]: I think you can simply say:

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  
  see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33)
```

**Comment by carlodri** [comment]
good job @nrbgt! I just need to figure out what to do with the renaming...

**Comment by carlodri** [comment]
Let's go with this PR, in-place rename is ok.... @nrbgt do you want to join me as a maintainer?

**Pull request merged**